### PR TITLE
Document the new Migrations class

### PR DESCRIPTION
**Pull request by HavokInspiration**
I'll take care of the French version once the english one is considered good.

### CODE REVIEW

**Comment by markstory** [comment]
Looks good to me.

**Comment by HavokInspiration** [comment]
@antograssiot @cake17 French translation added.

File: fr/migrations.rst [hunk 1]
```diff
@@ -338,3 +338,82 @@ installer. Toutes les commandes du plugin Migrations supportent l'option
...
+    $status = $migrations->status(['connection' => 'custom', 'source' => 'MyMigrationsFolder']);
+
+Vous pouvez passer n'importe quelle option que la commande de la console
+accepterait.
+La seule exception étant la commande ``markMigrated`` qui attend le numéro de
+version de la migration à marquer comme "migrée" comme premier argument. 
+Passez le tableau de paramètres en second argument pour cette méthode.
+
+En option, vous pouvez passer ces paramètres au constructeur de la classe.
+Ils seront utilisés comme paramètres par défaut et vous éviterons ainsi d'avoir
```
Comment (antograssiot) [inline fr/migrations.rst]: éviterez

File: fr/migrations.rst [hunk 2]
```diff
@@ -338,3 +338,82 @@ installer. Toutes les commandes du plugin Migrations supportent l'option
         bin/cake migrations status -p PluginName
 
         bin/cake migrations migrate -p PluginName
+
+Effectuer des Migrations en dehors d'un environnement Console
+=============================================================
+
+.. versionadded:: cakephp/migrations 1.2.0
+
+Depuis la sortie de la version 1.2 du plugins migrations, vous pouvez effectuer
```
Comment (antograssiot) [inline fr/migrations.rst]: du plugin

**Comment by cake17** [comment]
besides @antograssiot comments, looks good to me, nice work @HavokInspiration :+1:

**Comment by HavokInspiration** [comment]
Mistakes fixed

**Comment by antograssiot** [comment]
:clap:

**Comment by ravage84** [comment]
Made some small fixes in https://github.com/cakephp/docs/commit/045a337314a2a4f5cad42164d2876dd7a695b48c

**Comment by HavokInspiration** [comment]
Thanks @ravage84

**Pull request merged**